### PR TITLE
for review

### PR DESCRIPTION
**Pull request by eric-bel**

### CODE REVIEW

File: users.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import express from "express";
+
+const router = express.Router();
+
+const users = [
```
Comment (devbarden) [inline users.js:5]: please store the data in json files not in js

File: views/add-post.html [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Node.js Course</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap"
+      rel="stylesheet"
+    />
+    <style>
```
Comment (devbarden) [inline views/add-post.html:11]: Styling should keep in .css files

File: views/index.html [hunk 3]
```diff
@@ -1,31 +1,68 @@
...
-    <h1>Home page</h1>
-    <p>Hello it's a site home page</p>
-  </body>
-</html>
+<head>
+  <meta charset="UTF-8">
+  <title>Node.js Course</title>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+  <style>
```
Comment (devbarden) [inline views/index.html:8]: the same about styling

File: app.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+app.post('/', function (req, res) {
+    res.send('Got a POST request');
+});
+
+app.put('/:id', function (req, res) {
+    res.send('Got a PUT request');
+});
+
+app.delete('/:id', function (req, res) {
+    res.send('Got a DELETE request');
```
Comment (devbarden) [inline app.js:29]: why you aren't implement logic for endpoints ?

**Review by devbarden — COMMENTED** [review]
The main logic regarding endpoint not implemented. So please fix it

**Comment by devbarden** [comment]
you can fix this stuff in your next pr and this one you can merge if you need but you have currently conflicts in this PR so please fix it

File: app.js (hunk 4 above)
Comment (eric-bel) [reply app.js:29]: First, I want to thank you for the review)

All your comments will be considered and corrected.

At the expense of the logic of the endpoints, I agree, I didn’t write it down, time was running out, but now I’m working closely with this in authorization)

File: users.js (hunk 1 above)
Comment (eric-bel) [reply users.js:5]: First, I want to thank you for the review)

All your comments will be considered and corrected.

At the expense of the logic of the endpoints, I agree, I didn’t write it down, time was running out, but now I’m working closely with this in authorization)

**Comment by eric-bel** [comment]
First, I want to thank you for the review)

All your comments will be considered and corrected.

At the expense of the logic of the endpoints, I agree, I didn’t write it down, time was running out, but now I’m working closely with this in authorization)

Yes, did the merge, thank you very much, now I'm almost finishing my second homework and will do a PR

**Pull request merged**